tests/common/test_exception.py

```diff
@@ -1,46 +0,0 @@
-# Copyright 2019 Microsoft Corporation
```

Comment:
These tests enforce a contract that prevents adding parameters to AgentError exceptions, and they provide little value so I removed them.